docs/DEVELOPER-GUIDE.md

```diff
@@ -39,7 +43,8 @@
     * git remote add upstream git@github.com:appsody/appsody.git
 * set master to point to upstream
     * git branch master -u upstream/master
```

Comment:
If I run this on a clean setup following these instructions I get:
```
error: the requested upstream branch 'upstream/master' does not exist
hint:
hint: If you are planning on basing your work on an upstream
hint: branch that already exists at the remote, you may need to
hint: run "git fetch" to retrieve it.
hint: 
hint: If you are planning to push out a new local branch that
hint: will track its remote counterpart, you may want to use
hint: "git push -u" to set the upstream config as you push.```

Reply:
Do we need this level of detail in this guide? Surely we could link out to github.com for some of this?